### ISSUE TITLE
Cognitive Fatigue

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5927]
**Preferred term label:** Cognitive Fatigue

**Synonyms** Mental fatigue

**Definition (free text, please give PubMed ID)** Subjective decrease of mental energy which impacts cognitive performance., PMID:28237888 

**Parent term (use hpo.jax.org/app)** Cognitive impairment HP:0100543

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think this should be a subterm of Fatigue
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3058923/

**Comment by pnrobinson** [comment]
added term
